apps/cli/src/commands/build.ts

```diff
@@ -299,23 +299,23 @@ Update your application Dockerfile using one of the templates at https://github.
                 BuildApplication.createExt2Command(
                     bytes.parse(imageInfo.dataSize),
                 ),
-                SUNODO_DEFAULT_RETAR_TAR_PATH,
-                SUNODO_DEFAULT_EXT2_PATH,
+                gnuTarPath,
+                ext2Path,
             );
 
             // create machine snapshot
             if (!flags["skip-snapshot"]) {
                 await this.sdkRun(
                     sdkImage,
                     BuildApplication.createMachineSnapshotCommand(imageInfo),
-                    SUNODO_DEFAULT_EXT2_PATH,
-                    SUNODO_DEFAULT_MACHINE_SNAPSHOT_PATH,
+                    ext2Path,
+                    this.getContextPath("image"),
```

Comment:
For consistency, adding a `const snapshotPath = this.getContextPath("image");` at line 268 and then:

```suggestion
                    snapshotPath,
```